src/main/java/Duke.java

```diff
@@ -1,10 +1,34 @@
+import duke.backend.Storage;
+import duke.task.TaskList;
+import duke.frontend.Parser;
+import duke.frontend.Ui;
+
+import java.io.FileNotFoundException;
+
 public class Duke {
```

Comment:
Should there be header comments for all public classes and methods? (See Java coding standard -> Comments)

Reply:
Yeap! :) I haven't yet done the coding standards tag at the point of your review, but it's done now. Thank you for your comments!